From e7953aae7c5ab8205706f304f7f587419906210f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:44 -0400
Subject: [usb] usb_wwan: remove redundant modem-control request

Message-id: <1410887463-20674-453-git-send-email-dzickus@redhat.com>
Patchwork-id: 95241
O-Subject: [RHEL7 PATCH 452/591] USB: usb_wwan: remove redundant modem-control request
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 02803542b713060e6e05fcb88fa9258fd46985ca
Author: Johan Hovold <jhovold@gmail.com>
Date:   Mon May 26 19:23:21 2014 +0200

    USB: usb_wwan: remove redundant modem-control request

    The tty-port implementation has already made sure that DTR/RTS have been
    raised by calling dtr_rts so remove the redundant call from open.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/usb_wwan.c b/drivers/usb/serial/usb_wwan.c
index daaa5d7..9aeaccf 100644
--- a/drivers/usb/serial/usb_wwan.c
+++ b/drivers/usb/serial/usb_wwan.c
@@ -408,9 +408,6 @@ int usb_wwan_open(struct tty_struct *tty, struct usb_serial_port *port)
   }
  }
 
- if (intfdata->send_setup)
-  intfdata->send_setup(port);
-
  spin_lock_irq(&intfdata->susp_lock);
  portdata->opened = 1;
  if (++intfdata->open_ports == 1)
-- 
1.7.1